composer.json

```diff
@@ -52,6 +46,19 @@
         "post-autoload-dump": [
             "mkdir -p .Build/Web/typo3conf/ext/",
             "[ -L .Build/Web/typo3conf/ext/t3deploy ] || ln -snvf ../../../../. .Build/Web/typo3conf/ext/t3deploy"
+        ],
+        "test:unit": [
+            "[ -e '.Build/bin/phpunit' ] || composer update",
+            "TYPO3_PATH_ROOT=$(pwd)/.Build/Web .Build/bin/phpunit --colors -c .Build/vendor/nimut/testing-framework/res/Configuration/UnitTests.xml Tests/Unit"
+        ],
+        "test:functional": [
+            "[ -e '.Build/bin/phpunit' ] || composer update",
+            "TYPO3_PATH_ROOT=$(pwd)/.Build/Web .Build/bin/phpunit --colors -c .Build/vendor/nimut/testing-framework/res/Configuration/FunctionalTests.xml Tests/Functional"
+        ],
+        "test:params": [
```

Comment:
Please remove this block. this was only for PoC in the composer meeting.

Reply:
It has been removed now.